src/features/personDetails/PersonDetailsPage/Person/index.js

```diff
@@ -3,12 +3,13 @@ import DetailsTile from "../../../../common/DetailsTile";
 import MovieTiles from "../../../../common/MovieTiles";
 import Title from "../../../../common/Title";
 import { Wrapper } from "../../../../common/Wrapper";
-import { selectCredits, selectPersonDetails } from "../../personDetailsSlice";
+import { selectCredits, selectGenres, selectPersonDetails } from "../../personDetailsSlice";
```

Comment:
Aż dziwne, że szybciej tego nie wyłapaliśmy ;p

Reply:
Też się zdziwiłem, a przecież tyle razy na to patrzyłem 😅